react/features/base/toolbox/components/ToolboxItem.web.js

```diff
@@ -78,7 +82,8 @@ export default class ToolboxItem extends AbstractToolboxItem<Props> {
         };
 
         const elementType = showLabel ? 'li' : 'div';
-        const useTooltip = this.tooltip && this.tooltip.length > 0;
+        const useTooltip = Platform.OS === 'ios' && browser.isSafari()
```

Comment:
Please split condition to be more descriptive.
The logical operator is sufficient, please remove 'if' condition, it's not necessary in this case.

Reply:
The if condition is needed. What is not descriptive here?

Reply:
From my perspective the condition is too big and we can easily split to 2 parts which is checking the browser and the next one is checking the tooltip property and length. Please explain me why thew condition is needed.

Reply:
Because the platform and browser check is needed to determine/assign the false value in useTooltip. If we remove the if condition then the check of the platform and browser cannot be assigned on the variable useTooltip.